testsuite/tests/lib-smallint/test_int16.ml

```diff
@@ -1,7 +1,7 @@
 (* TEST
-   include stdlib_beta;
+   include stdlib_stable;
    modules = "test_smallint.ml";
-   flags = "-extension small_numbers_beta";
+   flags = "-extension small_numbers";
```

Comment:
It's fine to make these tests explicit by putting in `-extension small_numbers`, but there should also be a test somewhere that you get that for free (because it comes with `-extension-universe stable`, which is on by default).